test/helpers/AquaticLevels.js

```diff
@@ -173,7 +173,7 @@ module.exports = {
   },
   aquatic11: {
     isAquaticLevel: true,
-    ocean: "warm",
+    ocean: "cold",
```

Comment:
Unrelated - it's occurred to me that this file should probably be at `test/fixtures/AquaticLevels.js`

Reply:
It serves double-duty: we also use it for the levels in the playground!  https://code-dot-org.github.io/craft/?level=aquatic11